app/assets/stylesheets/application.scss

```diff
@@ -311,3 +312,20 @@ a {
     width: 80%;
   }
 }
+
+.user-profile {
+  height: 100%;
```

Comment:
Properties should be ordered background-color, border-color, border-style, border-width, height, padding-top, width